libbeat/scripts/Makefile

```diff
@@ -51,7 +51,7 @@ build: $(GOFILES)
 # Create test coverage binary
 .PHONY: buildbeat.test
 buildbeat.test: $(GOFILES)
-	go test -c -covermode=atomic -coverpkg ${GOPACKAGES_COMMA_SEP}
+	go test -c -race -covermode=atomic -coverpkg ${GOPACKAGES_COMMA_SEP}
```

Comment:
I will remove the change from this PR as it will be done in https://github.com/elastic/beats/pull/789